examples/causal_inference/interventional_distribution.ipynb

```diff
@@ -0,0 +1,1561 @@
+{
```

Comment:
<p>I'd add a bit of dramatic flair and advertise or hint at potential use-cases. We can see now why causal reasoning is both powerful and important for the implementation of policy and as an aid to counterfactual reasoning. Link back to the opening paragraph. Close with a strong recommendation that this distinction (intervention v conditional distributions needs to be part of the tool-kit of any working data scientist. </p><p> </p><p>Overall the post is great! This is such an important distinction and having the tools to make it well is huge! My comments above are minor stylistic points at most. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>Got it - good advice. Maybe not perfect, but I've given this a go</p>

Reply:
Looks great now!